contracts/test/strategies/ousd-metapool-ousd-tilted-pool.fork-test.js

```diff
@@ -22,24 +22,12 @@ describe("ForkTest: Convex 3pool/OUSD Meta Strategy - Titled to OUSD", function
       await mintTest(fixture, josh, usdt, "100000");
     });
 
-    describe("Mint", function () {
-      it("Should stake USDT in Curve gauge via metapool", async function () {
```

Comment:
Are we leaving the USDT mint test out?

Reply:
There's a duplicate test just below this line

Reply:
got it thanks